Makefile

```diff
@@ -10,11 +10,14 @@ VENV_DIR := .venv
 UV := uv
 PYTHON_VENV := $(VENV_DIR)/bin/python
 
-# Kafka configuration for local development
+# local development environment variables
```

Comment:
I believe these env variables may be better placed into your .envrc file, that way they are always loaded by direnv as long as you are in the launchpad directory